testsuite/ada_lsp/V628-024.symbol_diff/test.json

```diff
@@ -230,7 +230,15 @@
                }
             }
          },
-         "wait": []
+         "wait": [
+            {
+               "method": "textDocument/publishDiagnostics",
+               "params": {
+                  "uri": "file:///home/ogorodnik/adacore/tests/hello/src/main.adb",
```

Comment:
It it expected to have a full path here?

Reply:
Good catch, thank you